.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "third_party/zlib"]
     path = third_party/zlib
     url = https://github.com/madler/zlib.git
+[submodule "third_party/elfutils"]
+    path = third_party/elfutils
+    url = git://sourceware.org/git/elfutils.git
```

Comment:
@derekbruening Can we use an https version of the git repo? Nothing special, it's easier for users (like me) behind a http proxy.